pilot/pkg/networking/util/util.go

```diff
@@ -251,6 +251,9 @@ func GogoDurationToDuration(d *types.Duration) *duration.Duration {
 // Envoy computes a hash of RDS to see if things have changed - hash is affected by order of elements in the filter. Therefore
 // we sort virtual hosts by name before handing them back so the ordering is stable across HTTP Route Configs.
 func SortVirtualHosts(hosts []*route.VirtualHost) {
+	if len(hosts) < 2 {
```

Comment:
nit: len(hosts) == 1 is more readable than `< 2` check

Reply:
That isn't the same though, it could be an empty list